desktop/wallet/main.js

```diff
@@ -17,6 +18,12 @@ const isLinux = () => process.platform === 'linux';
 const isMacOS = () => process.platform === 'darwin';
 const isWindows = () => process.platform === 'win32';
 
+function handleLatestUpdate(newVersion) {
+  if (newVersion){
```

Comment:
Technically this is an anti-pattern but it's so small it doesn't matter.

Reply:
ty sir... you're right. but tbh I'm struggling with failing builds on travis due to OOM